pom.xml

```
@@ -96,31 +200,49 @@
-                <version>0.11.2</version>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <version>${surefire.version}</version>
+                <configuration>
+                    <argLine>-Djava.awt.headless=true</argLine>
+                </configuration>
+            </plugin>
+            <plugin>
+                <artifactId>maven-checkstyle-plugin</artifactId>
+                <version>2.17</version>
                 <executions>
                     <execution>
-                        <id>hub</id>
+                        <id>validate</id>
+                        <phase>validate</phase>
                         <goals>
-                            <goal>kafka-connect</goal>
+                            <goal>check</goal>
                         </goals>
                         <configuration>
-                            <confluentControlCenterIntegration>true</confluentControlCenterIntegration>
-                            <documentationUrl>https://jcustenborder.github.io/kafka-connect-documentation/projects/kafka-connect-transform-xml/</documentationUrl>
-                            <componentTypes>
-                                <componentType>transform</componentType>
-                            </componentTypes>
-                            <tags>
-                                <tag>Transform</tag>
-                                <tag>Xml</tag>
-                            </tags>
-                            <title>Xml Transformation</title>
-                            <supportUrl>${project.issueManagement.url}</supportUrl>
-                            <supportSummary>Support provided through community involvement.</supportSummary>
+                            <configLocation>checkstyle.xml</configLocation>
+                            <encoding>UTF-8</encoding>
+                            <consoleOutput>true</consoleOutput>
+                            <failsOnError>true</failsOnError>
+                            <includeResources>false</includeResources>
+                            <includeTestResources>false</includeTestResources>
+                            <sourceDirectory>${project.build.sourceDirectory}</sourceDirectory>
                         </configuration>
                     </execution>
                 </executions>
+                <dependencies>
+                    <dependency>
+                        <groupId>com.github.jcustenborder.kafka.connect</groupId>
+                        <artifactId>kafka-connect-checkstyle</artifactId>
+                        <version>1.0.0.0</version>
+                    </dependency>
+                </dependencies>
+            </plugin>
+            <plugin>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.5.1</version>
+                <inherited>true</inherited>
+                <configuration>
+                    <source>17</source>
+                    <target>17</target>
```

Use ${java.version} here
```suggestion
                    <source>${java.version}</source>
                    <target>${java.version}</target>
```